### ISSUE TITLE
Unnecessary reimplementation of `convertRect`

### ISSUE THREAD

**Issue by potmo** [teodorpatras/EasyTipView#53]
In the `arrange` function the `refViewFrame` is calculated with an extension to `UIView` called `originWithinDistantSuperView`.

In iOS 8 there is native support for that functionality (https://developer.apple.com/reference/uikit/uiview/1622442-convert)

```swift
let refViewFrame = CGRect(origin: presentingView!.originWithinDistantSuperView(superview), size: presentingView!.frame.size)
```
can be replaced by
```swift
let refViewFrame = presentingView!.convertRect(presentingView!.bounds, toView: superview)
```
(in swift 3 it has changed name to just `convert`)

**Comment by teodorpatras** [comment]
Hi, thanks for your input. That reimplementation is part of some older code I intend to remove. Did you have any problems with the positioning of the tooltip?

**Comment by potmo** [comment]
We had some problems positioning the tooltip if it was a child of a `UITableView` or `UIScrollView` and while investigating we at one time thought that the `originWithinDistantSuperView` had a bug and tested with replacing it with `convertRect`. We realised that that was not the problem but figured that it might be a good idea to replace it (since it was already implemented natively). Anyway the fix for our actual problem is in #52

**Comment by teodorpatras** [comment]
Got rid of the legacy method and released your PR in version `1.0.1`. Cheers!
